Standards/scs-XXXX-v1-volume-type-standard.md

```diff
@@ -0,0 +1,138 @@
+
+The parameters of volume types described in this standard do not have to be applied to the chosen default volume type. And this standard will not make any assumptions about parameters being present in the default volume type.
+
+## REQUIRED volume types
+
+Currently, this standard will not require volume types with certain specification. This will change in the future.
+
+## RECOMMENDED volume types
+
+This standard recommends to have one or more volume types, that satisfy the need for encrpytion and replication.
+
+### Encryption
+
+There SHOULD at least be one volume type with the encryption aspect.
+
+Encryption for volumes is an option which has to be configured within the volume type. As an admin it is possible to set encryption-provider, key size, cipher and control location. Additionally to be discoverable by users an admin has to edit the description and add `[scs:encrypted]` at the beginning or after another scs aspect. It should look like this example:
+
+```text
+openstack volume type show LUKS
++--------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Field              | Value                                                                                                                                                        |
++--------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| access_project_ids | None                                                                                                                                                         |
+| description        | [scs:encrypted] This volume uses LUKS-encryption                                                                                                             |
+| id                 | d63307fb-167a-4aa0-9066-66595ea9fb21                                                                                                                         |
+| is_public          | True                                                                                                                                                         |
+| name               | LUKS                                                                                                                                                         |
+| qos_specs_id       | None                                                                                                                                                         |
++--------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+### Replication
+
+There SHOULD at least be one volume type with the replication aspect.
```

Comment:
This sentence is redundant. The rest of the section belongs under a new section "Implemenation notes" (which should be a Supplement).